app/AccountancyModule/PaymentModule/presenters/PaymentPresenter.php

```diff
@@ -118,7 +119,12 @@ public function actionDefault(int $id): void
             $this['pairButton']->setGroups([$id]);
         }
 
-        $nextVS = $this->model->getNextVS($group->getId());
+        try {
+            $nextVS = $this->model->getNextVS($group->getId());
+        } catch (InvalidArgumentException $exception) {
+            $this->flashMessage('Nelze vygenerovat následující VS!', 'error');
```

Comment:
Nemůžeme použít specifičtější hlášku?

Víme, že ta situace nastane, pokud by vygenerování dalšího VS vygenerovalo neplatný VS (tedy VS o více než 10 znacích). To by bylo fajn uživateli říci.